src/haz3lweb/view/SchoolMode.re

```diff
@@ -11,6 +11,10 @@ type t = {
   grading_report: Grading.GradingReport.t,
 };
 
+/* Extracts and returns the results
```

Comment:
this function doesn't return the results and grades, it returns a SchoolMode.t value by computing the missing pieces from the given input